os-depends

```diff
@@ -33,7 +33,8 @@ gnome-shell (>= 3.22.3)
 gnupg
 # Could just be i386 amd64, but theoretically supports other arches
 grub2 [!armhf]
-grub-efi-amd64-image-signed [amd64] | grub-efi-amd64-image [i386]
+grub-efi-amd64-image-signed [!armhf]
+grub-efi-ia32-image [!armhf]
```

Comment:
Personally I'd use `[amd64 i386]` as a whitelist in case we ever add another architecture (e.g, `aarch64`). I see `intel-microcode` below this is doing the same thing.

Reply:
Sounds good. I'll fix `intel-microcode` in a different PR